web-ui/src/main/resources/catalog/views/default/templates/recordView/featurecatalogue.html

```diff
@@ -1,7 +1,34 @@
-<div data-ng-repeat="featureType in mdView.current.record.featureTypes">
-  <h3 data-translate="">featureAttributeTable</h3>
+<div
+  class="panel panel-default gn-margin-top width-50"
+  data-ng-show="mdView.current.record.featureTypes.length > 0"
```

Comment:
```suggestion
  data-ng-show="mdView.current.record.featureTypes.length > 1"
```
only display it if there is more than one table ?